include/uk/arch/paging.h

```diff
@@ -160,6 +160,19 @@ __paddr_t PT_Lx_PTE_PADDR(__pte_t pte, unsigned int lvl);
 #endif
 #endif /* !__ASSEMBLY__ */
 
+/**
+ * PT_Lx_PTE_SET_PADDR(pte, lvl, paddr)
+ *
+ * @param pte a page table entry from a page table at the given level
+ * @param lvl a page table level [0..PT_LEVELS - 1]
+ * @param paddr the physical address which to set in the PTE
+ *
+ * @return the PTE with the updated physical address
+ */
+#ifndef PT_Lx_PTE_SET_PADDR
```

Comment:
This has to be inside `!__ASSEMBLY__`  as `__pte_t` causes an undefined mnemonic compiler error when `paging.h` is included by `kvm/arm/bpt64.s`.